src/primitive.js

```diff
@@ -426,7 +426,14 @@ var _ = Mavo.Primitive = class Primitive extends Mavo.Node {
 					}
 
 					for (let primitive of nodes) {
-						primitive.originalEditorUpdated({force: true});
```

Comment:
Was this line removed accidentally or we genuinely don't need it anymore?

Reply:
My investigation shows that this line is the one that was added in [the first bad commit](https://github.com/mavoweb/mavo/commit/9f7876e32ff03b33b4f6cf2295c9d14aaacd149b) and caused the regression. The only thing I did was revert the previous version of this code block.

I think I understand why this line was present—we get the same result as when we have the two added lines. However, to add to this, we destroy the original editor observer and replace it with the observer of the clone of the original editor. It works fine if the element is not in the edit mode initially. But when it is, we clone the original editor before Mavo can update expressions and handle the original editor mutations.

Reply:
It looks like we need this line since, without it, we break another test with dynamic selects. I should continue working on a proper fix.